libfdo-data/src/test_common.rs

```diff
@@ -34,6 +34,27 @@ pub fn run_external(script: &str, args: &[&str]) -> Output {
         .join("test_scripts")
         .join(format!("{}.go", script));
 
+    match std::fs::remove_file(format!(
+        "{}/target/debug/libfdo_data.so.0",
+        std::env::var("PWD").unwrap()
+    )) {
+        Ok(_) => (),
+        Err(e) if e.kind() == std::io::ErrorKind::NotFound => (),
+        Err(e) => panic!("Failed to remove libfdo_data.so symlink {:?}", e)
+
+    }
+    std::os::unix::fs::symlink(
+        format!(
+            "{}/target/debug/libfdo_data.so",
+            std::env::var("PWD").unwrap()
+        ),
+        format!(
+            "{}/target/debug/libfdo_data.so.0",
```

Comment:
Maybe make sure that this `.0` is automatically provided by soname version?

Reply:
do we want to shell out to grab that? perhaps cargo has the env in tests?

Reply:
Cargo should be setting `CARGO_VERSION_MAJOR` as part of the build, even for tests I believe?

Reply:
oh yeah it works